.github/scripts/dev-build.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# run from the root dir .github/scripts/dev-build.sh 13
+# 13 as default in case we forget to pass the argument
+VERSION=${1:-13}
```

Comment:
I think it would be worth having the full version in here: `0.1.0-alpha.13`. So we don't to modify this script when we move on.